.github/workflows/integration.yml

```diff
@@ -0,0 +1,35 @@
+name: CI - Integration Test
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Git checkout
+        uses: actions/checkout@v2
+
+      - name: Install Node JS
+        uses: actions/setup-node@v1
+        with:
+          node-version: v20.16.0
+
+      - uses: actions/checkout@v3
+      - name: Set up JDK 17
+        uses: actions/setup-java@v3
+        with:
+          java-version: '17'
+          distribution: 'temurin'
+          cache: maven
+
+      - name: Run Integrations
+        env:
+          FLYWAY_PASSWORD: ${{ secrets.DB_PASSWORD }}
```

Comment:
Is setting up the flyway environment variables required for this script?

Reply:
It was needed for Migration tests. I think we still need all environment variables to run the tests

Reply:
Removed